DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -190,13 +488,23 @@ void DmlDispatchable::Initialize()
 
 void DmlDispatchable::Bind(const Bindings& bindings, uint32_t iteration)
 {
-    auto bindingProps = m_operatorCompiled->GetBindingProperties();
+    auto bindingProps = m_compiledOperator->GetBindingProperties();
 
     BindingData inputBindingData = {};
-    FillBindingData(m_desc.bindPoints.inputs, &m_initBindings, &bindings, inputBindingData);
-
     BindingData outputBindingData = {};
-    FillBindingData(m_desc.bindPoints.outputs, &m_initBindings, &bindings, outputBindingData);
+
+    auto bindPoints = GetBindPoints();
+
+    if (m_isSerializedGraph)
```

Comment:
Based on the comment on the header file, if we keep only variable for initBindings, then we can git rid of the if/else here.